### PR TITLE
Change get min ts function name

### PR DESCRIPTION
**Pull request by ystaticy**

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

**Comment by ystaticy** [comment]
@MyonKeminta  PTAL~

**Comment by ystaticy** [comment]
DNM, waiting for @MyonKeminta  to review

File: oracle/oracle.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ type Oracle interface {
 	GetExternalTimestamp(ctx context.Context) (uint64, error)
 	SetExternalTimestamp(ctx context.Context, ts uint64) error
 
-	GetMinTimestamp(ctx context.Context) (uint64, error)
+	GetAllTSOKeyspaceGroupMinTS(ctx context.Context) (uint64, error)
```
Comment (MyonKeminta) [inline oracle/oracle.go:62]: Consider explain the detailed purpose and behavior of this API in comments somewhere. Here can be the place.

**Review by MyonKeminta — COMMENTED** [review]
Rest LGTM

**Review by MyonKeminta — APPROVED** [review]

**Review by iosmanthus — APPROVED** [review]

**Pull request merged**